docs/specification.md

```diff
@@ -372,28 +385,58 @@ repeated.
 
 ## Recovery Workflows
 
+If every user were using gittuf and were performing each operation by
+generating all of the correct metadata, following the specification, etc., then
+the procedure for handling each situation is fairly straightforward. However,
+an important property of gittuf is to ensure that a malicious or erroneous
+party cannot make changes that impact the state of the repository in a negative
+manner. To address this, this section discusses how to handle situations where
+something has not gone according to protocol. The goal is to recover to a
+"known good" situation which does match the metadata which a set of valid
+gittuf clients would generate.
+
 ### Recovering from accidental changes and pushes
 
-There are several scenarios here. If a user makes changes locally and tries to
-push them to the blessed copy, it should be quite easy to detect and reject the
-changes. A pre-receive hook on the server can be employed to ensure the client
-is also sending valid metadata for the set of changes. If not, the operation
-must be terminated.
+A trusted user makes changes to a branch protected by gittuf policies. The user
```

Comment:
Needs set up to be clear about what is happening here.  Also, what a "trusted user" is, isn't clear.